core_tic_tac_toe/display.py

```diff
@@ -17,7 +21,13 @@ def print_board(self, moves):
             vertical_bars
         )
 
-        return board
+        self.io.print_message(board)
+
+    def print_invalid_move_message(self):
+        self.io.print_message("Invalid position, please try again!")
+
+    def get_player_input(self):
+        return self.io.get_player_input()
```

Comment:
The name `Display` suggest to me that you are showing/displaying something. It sounds like getting user input doesn't fit that description. What do you think?